Bindle/BindleClient.cs

```diff
@@ -7,7 +7,7 @@
 using System.Threading.Tasks;
 using Tomlyn.Model;
 
-namespace Bindle
+namespace Deislabs.Bindle
```

Comment:
Are we `Deislabs` or `DeisLabs`?  I had kind of assumed the latter but not sure.

Reply:
yeah, unfortunately I already used the former with wagi-dotnet, I can make a breaking change there (I am about to make other breaking changes) but it will be inconsistent with the naming in nuget as I used the same form there (https://www.nuget.org/packages/Deislabs.WAGI/0.6.0-preview)

Reply:
Stick with `Deislabs` then - it's not important enough to warrant going back and changing other things.